azimuth_capi/operator.py

```diff
@@ -481,6 +486,11 @@ async def on_cluster_create(logger, instance, name, namespace, patch, **kwargs):
     labels = patch.setdefault("metadata", {}).setdefault("labels", {})
     labels[f"{settings.api_group}/cluster-template"] = instance.spec.template_name
 
+    # Reset the timeout counter, now we have completed the update
+    # and we need to wait for all the components to do their updates
+    instance.status.last_updated_timestamp = dt.datetime.now(dt.timezone.utc)
+    await save_cluster_status(instance)
```

Comment:
Not sure we need to do this? I thought the idea was to have a timeout from the point that an update was made? The statement above accomplishes this just fine.

Reply:
So the idea is it can take quite a while to get to this statement, if you need to retry the above stuff waiting for the lease to go active, so I reset it here. Maybe that is overkill, it was useful in my odd testing edge cases.

Reply:
I remember what is was, it was for the case where we timed out in an error loop, then the operator fixes up the problem, and update finally completes, and at that point I reset the counter so you see progress in the UI again. Its an edge case for sure...